pool/web/view/component/button.ml

```diff
@@ -0,0 +1,8 @@
+open Tyxml.Html
+
+let add label path =
```

Comment:
I think this file can be removed

Reply:
ah, yes! I moved this as part of the other refactor. Will track down that `build_add_button` function replace it.

Reply:
fixed